sdk/src/Core/AwsXrayRecorderBuilder.cs

```diff
@@ -182,10 +182,10 @@ public AWSXRayRecorderBuilder WithPlugin(IPlugin plugin)
         }
 
         /// <summary>
-        /// Adds the given plugin to builder
+        /// Sets the address for the xray daemon.
```

Comment:
Wow. Never realized that the API and the doc were completely different.
Thanks so much for fixing these.

Reply:
@srprash, no problems.  I'm sure you've got a lot going on with the XRay Team currently.  Are you able to give any updates yet on when we can expect a new release?  On a separate PR, I've pushed out a non-breaking form of support for net60.  It would be beneficial for a product I'm working on if we could get a release.  We're currently blocked from adopting net60 until we get the interceptor updates, and the current released assemblies reference unnecessary packages for net31 which contain security vulnerabilities.  Those issues have been fixed for months, but we haven't had a release to take advantage of those fixes.

Reply:
@jon-armen I would love to get the new version released and have you (and others) unblocked. It's just that we are refactoring our build system and figuring out some technicalities on our end.
This is still a high priority for us and will get the new release out ASAP. Unfortunately, I can't disclose the ETA yet.